main_2024.py

```diff
@@ -357,24 +396,16 @@ def main() -> int:
                 return -1
 
         try:
-            geolocation_data = geolocation_to_main_queue.queue.get_nowait()
+            cluster_estimations = cluster_estimation_to_main_queue.queue.get_nowait()
         except queue.Empty:
-            geolocation_data = None
-
-        if geolocation_data is not None:
-            for detection_world in geolocation_data:
-                main_logger.debug("Detection in world:", True)
-                main_logger.debug(
-                    "geolocation vertices: " + str(detection_world.vertices.tolist()), True
-                )
-                main_logger.debug(
-                    "geolocation centre: " + str(detection_world.centre.tolist()), True
-                )
-                main_logger.debug("geolocation label: " + str(detection_world.label), True)
-                main_logger.debug(
-                    "geolocation confidence: " + str(detection_world.confidence), True
-                )
-
+            cluster_estimations = None
+
+        if cluster_estimations is not None:
+            for cluster in cluster_estimations:
+                main_logger.debug("Cluser in world: True")
+                main_logger.debug("Cluster location x: " + str(cluster.location_x))
+                main_logger.debug("Cluster location y: " + str(cluster.location_y))
+                main_logger.debug("Cluster spherical variance: " + str(cluster.spherical_variance))
```

Comment:
These are all missing the second parameter of debug set to `True`.

Reply:
True by default